src/pages/ImportData.js

```diff
@@ -27,6 +27,7 @@ class Importer extends Component {
           div({}, [
             h(WorkspaceSelector, {
               authorizationDomain: ad,
+              filter: ({ accessLevel }) => Utils.canWrite(accessLevel),
```

Comment:
`filter: Utils.canWrite` ?

Reply:
We'd have to make that function accept a workspace object. Which we could do, I thought about it.

Reply:
Ah, right, nvm.